ATP.DataAccessLayer.Test/RepositoryTest/FlightRepositoryTest.cs

```diff
@@ -1,86 +1,82 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Globalization;
 using ATP.BusinessLogicLayer.Models;
-using ATP.DataAccessLayer.Repository;
-using Moq;
-using Xunit;
+using ATP.DataAccessLayer.Mapper;
+using ATP.DataAccessLayer.Models;
+using CsvHelper;
+using Microsoft.Extensions.Logging;
 
-namespace ATP.DataAccessLayerTest.RepositoryTest
+namespace ATP.DataAccessLayer.Repository;
+
+public class FlightRepository
 {
-    public class FlightRepositoryTest
+    private List<FlightDomainModel> _flights;
+    private readonly string _csvFilePath;
+    private FlightMapper _mapper;
+    private readonly ILogger<FlightRepository> _logger;
+
+    public FlightRepository(string csvFilePath, FlightMapper mapper, ILogger<FlightRepository> logger)
     {
-        private readonly Mock<IFlightRepository> _repositoryMock = new Mock<IFlightRepository>();
-        private readonly List<FlightDomainModel> _flights;
+        _csvFilePath = csvFilePath;
+        _mapper = mapper;
+        _flights = LoadFlightsFromCsv(csvFilePath);
+        _logger = logger;
+    }
 
-        public FlightRepositoryTest()
+    private List<FlightDomainModel> LoadFlightsFromCsv(string csvFilePath)
+    {
+        try
         {
-            _flights = new List<FlightDomainModel>
+            using (var reader = new StreamReader(csvFilePath))
+            using (var csv = new CsvReader(reader, CultureInfo.InvariantCulture))
```

Comment:
For `InvariantCulture`, you can add something on the csproj file like `<InvariantGlobalization>false`<\InvariantGlobalization>` 
```suggestion
            using var csv = new CsvReader(reader, CultureInfo.InvariantCulture);
```